src/sage/schemes/elliptic_curves/ell_point.py

```diff
@@ -3871,68 +3928,73 @@ def order(self):
+            sage: P.order()
             1427247692705959881058262545272474300628281448
-            sage: P.order() == E.cardinality()                                          # optional - sage.rings.finite_rings
+            sage: P.order() == E.cardinality()
             True
 
         The next example has `j(E)=0`::
 
+            sage: # needs sage.rings.finite_rings
             sage: p = 33554501
-            sage: F.<u> = GF((p,2))                                                     # optional - sage.rings.finite_rings
-            sage: E = EllipticCurve(F, [0,1])                                           # optional - sage.rings.finite_rings
-            sage: E.j_invariant()                                                       # optional - sage.rings.finite_rings
+            sage: F.<u> = GF((p,2))
+            sage: E = EllipticCurve(F, [0,1])
+            sage: E.j_invariant()
             0
-            sage: P = E.random_point()                                                  # optional - sage.rings.finite_rings
-            sage: P.order() # random                                                    # optional - sage.rings.finite_rings
+            sage: P = E.random_point()
+            sage: P.order()  # random
             16777251
 
         Similarly when `j(E)=1728`::
 
+            sage: # needs sage.rings.finite_rings
             sage: p = 33554473
-            sage: F.<u> = GF((p,2))                                                     # optional - sage.rings.finite_rings
-            sage: E = EllipticCurve(F, [1,0])                                           # optional - sage.rings.finite_rings
-            sage: E.j_invariant()                                                       # optional - sage.rings.finite_rings
+            sage: F.<u> = GF((p,2))
+            sage: E = EllipticCurve(F, [1,0])
+            sage: E.j_invariant()
             1728
-            sage: P = E.random_point()                                                  # optional - sage.rings.finite_rings
-            sage: P.order() # random                                                    # optional - sage.rings.finite_rings
+            sage: P = E.random_point()
+            sage: P.order()  # random
             46912611635760
 
         TESTS:
 
         Check that the order actually gets cached (:trac:`32786`)::
 
-            sage: E = EllipticCurve(GF(31337), [42,1])                                  # optional - sage.rings.finite_rings
-            sage: P = E.lift_x(1)                                                       # optional - sage.rings.finite_rings
-            sage: hasattr(P, '_order')                                                  # optional - sage.rings.finite_rings
+            sage: # needs sage.rings.finite_rings
+            sage: E = EllipticCurve(GF(31337), [42,1])
+            sage: P = E.lift_x(1)
```

Comment:
So all prime finite fields are "small"... I thought there is a limit...

Reply:
No, there's the block tag because it's not small.

Reply:
Ah, I see. I missed it.